### DISCUSSION TITLE
which frontpanel would be the right ?

### DISCUSSION THREAD

**Discussion by Her0-1337** [WLAN-Pi/feedback#15] [Hardware - WLAN Pi R4] [answered]
Hi all,

I am new to the WLAN-Pi project and heared about it 3 weeks ago. Since this time I was on the search for a NanoPi neo2 but couldn't found one to buy.
Now I am glad that version 3 runs on a normal RaspberryPi cause I got a Pi 3B+ in my drawer. 
Could you tell me which which OLED display would be the correct one which works with the Raspberry ?
Sorry for the n00b question this all is new for me.

Thanks in advance!

**Comment by jolla** [comment, marked as answer]
Nothing official at this time, but prior to getting our hands on the WLAN Pi Pro hardware some of the developers had good luck using this hat https://www.amazon.com/waveshare-1-44inch-Interface-Direct-pluggable-Raspberry/dp/B077Z7DWW1

**Comment by Her0-1337** [comment]
Orderd this HAT but unfortunately I can't get it running :(
Even not with the guides on the manufacturer page.

**Reply by jiribrejcha** [reply]
It requires the latest software. Let us see if we can get it fully tested and released in the coming week or so. 

Does the display backlight light up when you power on the Pi?

Here is the latest on Community Edition hardware: https://userguide.wlanpi.com/v/wlan-pi-community-edition/guides/supported-hardware

And supported features: https://userguide.wlanpi.com/v/wlan-pi-community-edition/guides/supported-features

**Reply by Her0-1337** [reply]
![IMG_2186(1)](https://user-images.githubusercontent.com/101107893/158962421-4bfb9ae8-b3c4-4162-9b9a-e630c1be231c.JPEG)


Yes it light up

**Reply by jiribrejcha** [reply]
Perfect. That's exactly what I would expect when running the current image.

It is just a matter of getting the new wlanpi-fpms package to you. We just need to test it before releasing. Stay tuned, please.

**Comment by Her0-1337** [comment]
Guys, you are the best, great work!
Seems like I need a 3D printer now :D


![IMG_2193](https://user-images.githubusercontent.com/101107893/159114131-78b8e811-fb49-4692-9a61-1f96b649bb9e.JPEG)

**Reply by jiribrejcha** [reply]
Our pleasure 👍🏻 

We pushed the code that supports Community Edition last night, and I see you have already discovered that 😉

For others who might be reading this later:
- Community Edition based on Raspberry Pi 4 and Waveshare 1.44” LCD Display HAT with buttons now works after you upgrade WLAN Pi software packages to their latest version. 
- To update the packages, simply SSH into your WLAN Pi (default credentuals are wlanpi/wlanpi) and execute:
`sudo apt update && sudo apt upgrade`

**Reply by Her0-1337** [reply]
Hey there,
it's now nearly 3 years ago, I just installed WLAN-Pi v3.3.0.
This time on an raspberry Pi 5 but the Display isn't working anymore.
The OS itself is running fine, am I missing something?

Edit: Ahh just saw the release note, that RPI5 is not supported :(
